DEVELOPMENT.md

```diff
@@ -295,41 +389,18 @@ BentoML also provides git hooks that developers can install with:
 make hooks
 ```
 ## Stubs
-Refers to [Installation](https://github.com/microsoft/pyright#installation) to install pyright correctly.
+Refer to [Installation](https://github.com/microsoft/pyright#installation) to install pyright correctly.
 
-In order to make pyright function correctly one also need to run the following scripts alongside with the stubs provided
- in the main repository.
+In order for pyright to function correctly you must also run the following scripts alongside the stubs provided in the main repository.
 
-One can also clone a [copy](https://github.com/bentoml/stubs) of all dependencies stubs used by BentoML to `typings/` via:
+You can also clone a [copy](https://github.com/bentoml/stubs) of all the dependencies stubs used by BentoML to `typings/` via:
 ```bash
 # Assuming at $GIT_ROOT
 git clone git@github.com:bentoml/stubs.git
 \rm -rf typings/.git* typings/*.{sh,toml,txt,cfg,py,md}
 ```
 
-## Creating Pull Request on GitHub
-
-[Fork BentoML project](https://github.com/bentoml/BentoML/fork) on GitHub and
-add upstream remotes to local BentoML clone:
-
-```bash
-git remote add upstream git@github.com:bentoml/BentoML.git
-```
-
-Make the changes either to fix a known issue or adding new feature
-
-In order for us to manage PR and Issues systematically, we encourage developers to use hierarchical branch folders to manage branch naming.
-Run `./dev/install_git_hooks.sh` to install `pre-commit` hooks. We will check if your branch naming
-follows the given regex : `^(feature|bugfix|improv|lib|prerelease|release|hotfix)\/[a-zA-Z0-9._-]+$`. This
-is partially based on how [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/repos/git/require-branch-folders?view=azure-devops&tabs=browser)
-manages its repos and conventional commits spec (see below for more information):
-- feature: new features/proposals users want to integrate into the library, not a new feature for a build script
-- bugfix: bugfix of a feature, not a fix to a build script
-- improv: improvements/refactor/cleanup production code, eg. reformat, pylint, etc.
-- lib: related to internal libraries, features required by the production code
-- prerelease: alpha/beta features that might should be included in the prerelease of the library. This would help testing new features/integrations for the library
-- release: included all features that is production ready
-- hotifx: patch of current bugs in production code
```

Comment:
can you restore this part, we are putting this here since we want to follow conventional commits. Refers to [here](https://www.conventionalcommits.org/en/v1.0.0/)

Reply:
My opinion on conventional commits: with our workflow they're kind of useless.

Branch hygiene is kind of meaningless because it's not our problem; we don't develop in a monolithic repo (thank goodness) so users can do branches however they like.

Because we're squash-and-merging, commits on people's branches are also kind of meaningless. Commit hygiene in general is a bit lackluster overall, and enforcing these arbitrary rules aren't actually helping. We should just make an effort to write sensible commit titles and messages when we squash-and-merge; I think that's generally the better way to handle this kind of thing anyway. We're still asking users to title their PRs in a nice way, which should get us a decent amount of the way there.